Description: Fix lintian issue "W: libjpeg-turbo-progs: groff-message usr/share/man/man1/jpegtran.1.gz (line 1) 170: warning: macro 'f'' not defined"
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/jpegtran.1
+++ b/jpegtran.1
@@ -167,7 +167,7 @@
 to the nearest iMCU boundary (the lower right corner is unchanged.)  Thus, the
 output image covers at least the requested region, but it may cover more.  The
 adjustment of the region dimensions may be optionally disabled by attaching an
-'f' character ("force") to the width or height number.
+\'f\' character ("force") to the width or height number.
 
 The image can be losslessly cropped by giving the switch:
 .TP
